### DISCUSSION TITLE
What is the best way to run multiple dataproc templates in parallel?

### DISCUSSION THREAD

**Discussion by sjlva** [GoogleCloudPlatform/dataproc-templates#279] [Q&A] [answered]
If you have, for example, 100 JDBC tables and need to copy them to GCS. 
What would be the best approach to execute the same template, but with different parameters in parallel?

Can I assign multiple batches to a single DataProc Serverless cluster?

**Comment by NiloFreitas** [comment, marked as answer]
You can submit dataproc serverless batches in parallel, as shown here: https://github.com/GoogleCloudPlatform/dataproc-templates/tree/main/notebooks
